### ISSUE TITLE
Consensus client error with `--eth1-endpoints` flag

### ISSUE THREAD

**Issue by mmsaki** [NethermindEth/sedge#354]
**Describe the bug**
`docker-compose.yml` script does not successfully start consensus client, I get the following error:

```shell
sedge-consensus-client  | error: Found argument '--eth1-endpoints' which wasn't expected, or isn't valid in this context
sedge-consensus-client  | 	Did you mean --eth1?
sedge-consensus-client  | 
sedge-consensus-client  | USAGE:
sedge-consensus-client  |     lighthouse beacon_node --datadir <DIR> --disable-upnp --eth1 --execution-endpoint <EXECUTION-ENDPOINT> --execution-jwt <EXECUTION-JWT> --http-address <ADDRESS> --http-port <PORT> --network <network> --port <PORT> --target-peers <target-peers> <--http|--gui|--staking>
``` 

**To Reproduce**
1. Install latest version of Sedge 
```shell
curl -L https://github.com/NethermindEth/sedge/releases/download/v1.3.0/sedge-v1.3.0-darwin-amd64 --output sedge
chmod +x ./sedge
mv ./sedge /usr/local/bin/sedge
``` 
2. Generate `docker-compose.yml` and `.env` files
```shell
sedge generate full-node --latest --network sepolia
``` 
3. Execute the setup
```shell
docker compose up -d
``` 
```shell
docker compose logs consensus
``` 

**Expected behavior**
The Lighthouse client should log that it started successfully

**Screenshots**
<img width="1676" alt="Screenshot 2024-02-02 at 7 20 13 PM" src="https://github.com/NethermindEth/sedge/assets/98189596/73a20178-b5f9-4b5f-952c-351adf502479">

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 13.6.3

**Additional context**
- sedge v1.3.0

**Comment by mmsaki** [comment]
Hello @stdevMac @cbermudez97. Could you please take a moment to review my pull request? I'd greatly appreciate your insights and feedback.

**Comment by adriantpaez** [comment]
Hi, @mmsaki thank you for the report and the open PR. This issue was fixed with the new version 1.3.1. Please try with the new version.

**Comment by mmsaki** [comment]
> Hi, @mmsaki thank you for the report and the open PR. This issue was fixed with the new version 1.3.1. Please try with the new version.

Awesome, thanks, that was a quick fix!
